### PR TITLE
Issue #11: Remove panels support from Backdrop version of Radix.

### PR DESCRIPTION
**Pull request by jenlampton**
Fixes https://github.com/backdrop-contrib/radix/issues/11

### CODE REVIEW

File: kits/default/assets/sass/global/_variable.scss [hunk 1]
```diff
@@ -506,38 +506,6 @@ $list-group-link-color:          #555 !default;
 $list-group-link-heading-color:  #333 !default;
```
Comment (shadcn) [inline kits/default/assets/sass/global/_variable.scss:506]: These Sass variables are required. They are part of Bootstrap panels and not Drupal panels. See http://getbootstrap.com/components/#panels. Thanks.

File: includes/form.inc [hunk 2]
```diff
@@ -295,30 +295,3 @@ function radix_date($variables) {
 
   return '<div' . backdrop_attributes($attributes) . '>' . backdrop_render_children($element) . '</div>';
 }
-
-/**
- * Implements theme_fieldset().
- */
-function radix_fieldset($variables) {
```
Comment (shadcn) [inline includes/form.inc:302]: This theme function is required (unless theme_fieldset is not required by Backdrop anymore?).

This renders fieldsets as Bootstrap Panels. See http://getbootstrap.com/components/#panels. Thanks.

File: kits/default/assets/sass/global/_variable.scss (hunk 1 above)
Comment (jenlampton) [inline kits/default/assets/sass/global/_variable.scss:506]: Ah, whoopsie.  putting these back...

File: includes/form.inc (hunk 2 above)
Comment (jenlampton) [inline includes/form.inc:302]: ah.  putting his back too...

**Comment by jenlampton** [comment]
okay - just updated the commit.  So much for my find-replace on the word panel! :)

**Pull request merged**